packages/contracts-bedrock/scripts/deploy/DeployImplementations.s.sol

```diff
@@ -719,9 +719,6 @@ contract DeployImplementations is Script {
                 _input.faultGameV2MaxClockDuration >= _input.faultGameV2ClockExtension,
                 "DeployImplementations: maxClockDuration must be >= clockExtension"
             );
-            require(
-                _input.faultGameV2ClockExtension > 0, "DeployImplementations: faultGameV2ClockExtension must be > 0"
-            );
```

Comment:
is this a typo? We should either validate all parameters or don't.

Reply:
Originally this seemed to be required because we set clock extension to 0 for `fast` games used in op-e2e tests.  But since I've rebased it isn't needed - not sure if I was just confused earlier or if something else changed (I know that check had to be removed from the script to deploy additional dispute games but not sure why this check started failing).  Either way, it's back now.

Reply:
We could also have the fast game provide a non-zero dummy clock extension value.

Reply:
If you provide a non-zero value you have to also increase the clock duration (I think) because the clock duration has to be > clock extension.

Reply:
I think I found the right balance to keep this check in DeployImplementations but still have it as 0 for fast games. Even with v2 the fast games were being deployed with DeployDisputeGame which doesn't enforce this but elsewhere we were still setting it to 0 and the error messages aren't very clear about what's failing.

Glad you pushed back on this. :)